src/components/UserMapMarker.css

```diff
@@ -1,7 +1,5 @@
-.Event-dot {
-  width: '15px';
-  height: '15px';
-  background-color: #fc6888;
-  borderRadius: '10px';
-  box-shadow: 8px 8px rgba(255, 255, 255, 0.4)
+.User-dot {
+  width: 17px;
+  height: 50px;
+  background: url(http://me.blackwall.co/style/monogramBlack.png) no-repeat center center;
```

Comment:
Always download such images so that we have it in this project. Also quite important as we need to be able to run over HTTPS and then all linked content also has to be HTTPS.